### PR TITLE
update @swapr/ui lib to v0.1.4

### PR DESCRIPTION
**Pull request by Diogomartf**
**Add new code icon**
image

**Latest screenshots**
image
image
image

**accommodate styles to new `Dialog`**
image
image
image
**mobile**
image
image

**some problems on the "old"**
image
**bad padding x space**
image

### CODE REVIEW

**Comment by berteotti** [comment]
We should wait for the new release to fix the dialog close button position, right?

**Comment by Diogomartf** [comment]
> We should wait for the new release to fix the dialog close button position, right?

No, we don't have to wait, will just need to update the package, nothing else.

**Comment by Diogomartf** [comment]
> We should wait for the new release to fix the dialog close button position, right?

Updated!

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "@duneanalytics/client-sdk": "^0.2.1",
     "@fvictorio/newton-raphson-method": "^1.0.5",
     "@radix-ui/react-scroll-area": "^1.0.5",
-    "@swapr/ui": "^0.1.3",
+    "@swapr/ui": "^0.1.41",
```
Comment (berteotti) [inline package.json:16]: Why is it 0.1.41? 😅 Shouldn't be 0.1.4 ?

**Review by berteotti — APPROVED** [review]

**Review by ElRodrigote — APPROVED** [review]
Love it!

File: package.json (hunk 1 above)
Comment (Diogomartf) [reply package.json:16]: Nope, it's the newest version.

**Pull request merged**